### DISCUSSION TITLE
NVIDIA GeForce RTX 3060 12G显存，运行量化的cli_demo.py，提示内存不足。

### DISCUSSION THREAD

**Discussion by spar025** [zai-org/ChatGLM3#1159] [Q&A] [answered]
新人求助：
       当前配置运行量化的chatGLM3-6B，跑不起来，不是说6G显存就可以吗？我要怎么做，才能跑起来呢？

问题描述：
       1、CPU及内存
         
![image](https://github.com/THUDM/ChatGLM3/assets/33474854/3790db53-b3a4-44ef-8b5c-52cd422ff52f)

       2、显卡
![image](https://github.com/THUDM/ChatGLM3/assets/33474854/c29f932f-d4a6-49aa-8dbd-abbf0ede1ead)

![image](https://github.com/THUDM/ChatGLM3/assets/33474854/ad09a538-5fe5-4011-adf7-0b39548c7d79)
       3、私有化部署修改cli_demo.py
            
![image](https://github.com/THUDM/ChatGLM3/assets/33474854/bfe971af-5656-4a72-a603-8acc04de028c)
       4、运行cli_demo.py
         
![image](https://github.com/THUDM/ChatGLM3/assets/33474854/79e86522-d6bc-42a0-8458-d1323283b3e9)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
hf模型更新了吗，应该跑得起来的

**Reply by spar025** [reply]
跑起来了，多了个参数，device_map，删了就好了

**Reply by SHENGYUKing** [reply]
我这边遇到的问题还不太一样，我从modelscope上下载了chatglm3-6b（不是base）并成功在本地使用RTX3080采用INT4方案跑起来了，但是回答内容几乎没有任何逻辑、近乎乱码，请问遇到过类似情况吗？
![5c5551925151fcdaa7cc3a7a70c9004](https://github.com/THUDM/ChatGLM3/assets/35353285/30d5aa92-225c-4861-8981-133df2b451b5)
![a204a253d6383886810b31abb37b149](https://github.com/THUDM/ChatGLM3/assets/35353285/2d91baa8-041d-4b74-a17f-81cac06f2c59)
